snow/consensus/snowman/topological.go

```diff
@@ -23,7 +22,9 @@ import (
 )
 
 var (
-	errDuplicateAdd = errors.New("duplicate block add")
+	errDuplicateAdd            = errors.New("duplicate block add")
+	errTooManyProcessingBlocks = errors.New("too many processing blocks")
+	errBlockProcessingTooLong  = errors.New("block processing too long")
 
 	_ Factory   = (*TopologicalFactory)(nil)
```

Comment:
(not related to this PR) should we move these checks above the errors?

Reply:
🤷